### PR TITLE
Avoid NPE when misusing library import

### PR DESCRIPTION
**Pull request by adamretter**
Closes https://github.com/eXist-db/exist/issues/1245

### CODE REVIEW

**Comment by dizzzz** [comment]
no test?

File: src/org/exist/xquery/functions/inspect/InspectModule.java [hunk 1]
```diff
@@ -11,52 +11,54 @@
...
             new FunctionReturnSequenceType(Type.ELEMENT, Cardinality.ZERO_OR_ONE,
-                "An XML fragment describing the module and all functions contained in it.")
-        );
-        
-        public final static FunctionSignature FNS_INSPECT_MODULE_URI = new FunctionSignature(
+                    "An XML fragment describing the module and all functions contained in it.")
+    );
+
+    public final static FunctionSignature FNS_INSPECT_MODULE_URI = new FunctionSignature(
             new QName("inspect-module-uri", InspectionModule.NAMESPACE_URI, InspectionModule.PREFIX),
```
Comment (shabanovd) [inline src/org/exist/xquery/functions/inspect/InspectModule.java]: misleading function name?

File: src/org/exist/xquery/functions/inspect/InspectModule.java [hunk 2]
```diff
@@ -11,52 +11,54 @@
...
             "Returns an XML fragment describing the " +
-            "module identified by the given URI and the functions/variables contained in it.",
-            new SequenceType[] {
-                new FunctionParameterSequenceType("uri", Type.ANY_URI, Cardinality.EXACTLY_ONE,
-                    "The location URI of the module to inspect"),
+                    "library module identified by the given namespace URI and the functions/variables contained in it.",
+            new SequenceType[]{
+                    new FunctionParameterSequenceType("uri", Type.ANY_URI, Cardinality.EXACTLY_ONE,
```
Comment (shabanovd) [inline src/org/exist/xquery/functions/inspect/InspectModule.java]: change name to `namespace`?

File: src/org/exist/xquery/functions/inspect/InspectModule.java (hunk 1 above)
Comment (adamretter) [inline src/org/exist/xquery/functions/inspect/InspectModule.java]: I do not want to change the function name as that would mean an API change.

File: src/org/exist/xquery/functions/inspect/InspectModule.java (hunk 2 above)
Comment (adamretter) [inline src/org/exist/xquery/functions/inspect/InspectModule.java]: No, this loads a module from the module location hint.

File: src/org/exist/xquery/functions/inspect/InspectModule.java (hunk 1 above)
Comment (shabanovd) [inline src/org/exist/xquery/functions/inspect/InspectModule.java]: Introduce new one and mark this as deprecated?

File: src/org/exist/xquery/functions/inspect/InspectModule.java (hunk 2 above)
Comment (shabanovd) [inline src/org/exist/xquery/functions/inspect/InspectModule.java]: and confuse come in ...
`importModule(args[0].getStringValue(), null, null)` -> `importModule( String namespaceURI, String prefix, String location )`

and also you change description to point that it's `namespace URI`.

File: src/org/exist/xquery/functions/inspect/InspectModule.java (hunk 1 above)
Comment (adamretter) [inline src/org/exist/xquery/functions/inspect/InspectModule.java]: Not the purpose of this PR.

**Comment by adamretter** [comment]
@dizzzz I just added the tests as requested.

**Pull request merged**